tests/Misc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Tests;
+
+use NunoMaduro\LaravelMojito\InteractsWithViews;
+use PHPUnit\Framework\AssertionFailedError;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
+ */
+final class Misc extends TestCase
```

Comment:
Can we improve the name of this class?

Reply:
No problem. I moved that test into an existing class.
To be fair, it can't also be removed. I added just to make sure I could replicate and fix the issue but, now that we don't use sprintf anymore, % is a normal character like many others so it doesn't make much sense to test it in the future.
My plan was to take it out again once we are happy with the feature.